docs/adr/0001-compatibility-with-different-play-version.md

```diff
@@ -0,0 +1,56 @@
+# Making the sbt-accessibility-plugin Compatible with various Play Framework versions
+
+* Status: proposed
+* Deciders: PlatUI
+* Date: 29 Jan 2024
+
+Technical Story: [PLATUI-2755](https://jira.tools.tax.service.gov.uk/browse/PLATUI-2755)
+
+## Context and Problem Statement
+
+How can we make the future versions of the sbt-accessibility-plugin compatible with different versions of the Play Framework? We need a solution that balances ease of implementation, maintainability, and compatibility.
+
+## Decision Drivers
+
+* Compatibility with multiple versions of Play Framework
+* Maintainability of the plugin
+* Ease of implementation
+* Future-proofing against new Play Framework versions
+
+## Considered Options
+
+* Option 1: Build a specific version of the `sbt-accessibility-linter` plugin for each Play Framework version.
+* Option 2: Use reflection in the `sbt-accessibility-linter` to determine the consuming project's Play version and dynamically include the correct `scalatest-accessibility-linter` library.
+* Option 3: Employ macros to determine the Play version at compile time and include the appropriate `scalatest-accessibility-linter` library.
+* Option 4: Decouple the `scalatest-accessibility-linter` library from the `sbt-accessibility-linter` plugin.
```

Comment:
```suggestion
* Option 4: Decouple the `scalatest-accessibility-linter` library from the `sbt-accessibility-linter` plugin.
* Option 5: Only support one version of play at a time - so future updates for old play versions would have to be done as patch versions
```
I think in the future I'd probably say do option 5 if we're not sure about future of something - like we did for play-ui